amps/ags/rm-community/rm-community-repo/unit-test/java/org/alfresco/module/org_alfresco_module_rm/job/DispositionLifecycleJobExecuterUnitTest.java

```diff
@@ -235,6 +235,9 @@ public void dispositionActionProcessed()
         doReturn(CUTOFF).when(mockedNodeService).getProperty(node1, RecordsManagementModel.PROP_DISPOSITION_ACTION);
         doReturn(RETAIN).when(mockedNodeService).getProperty(node2, RecordsManagementModel.PROP_DISPOSITION_ACTION);
         doReturn(parentAssoc).when(mockedNodeService).getPrimaryParent(any(NodeRef.class));
+        doReturn(false).when(mockedRecordFolderService).isRecordFolder(parentAssoc.getParentRef());
+        doReturn(true).when(mockedRecordService).isRecord(parentAssoc.getParentRef());
+        doReturn(false).when(mockedFreezeService).isFrozen(parentAssoc.getParentRef());
```

Comment:
This is fine to leave (for consistency with the rest of the file), but I wanted to say that the Mockito statements seem unnecessarily backwards here.